auth-api/src/auth_api/models/org.py

```diff
@@ -72,6 +72,18 @@ def find_by_org_id(cls, org_id):
         """Find an Org instance that matches the provided id."""
         return cls.query.filter_by(id=org_id).first()
 
+    @classmethod
+    def search_org(cls, org_type, name, status):
+        """Find all orgs with the given type."""
+        queries = []
+        if org_type:
```

Comment:
rename this to access_type if possible, and add org_type for BASIC|PREMIUM ?

Reply:
sure.will do thus when I test the frontend

Reply:
actually did the access_type.org_Type will add later